EventProcessor/EventProcessor.WebJob/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Autofac;
+using Microsoft.Azure.Devices.Applications.RemoteMonitoring.EventProcessor.WebJob.Processors;
+using Microsoft.Azure.IoT.Samples.EventProcessor.WebJob.Processors;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.EventProcessor.WebJob
+{
+    public static class Program
+    {
+        static CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
+        static IContainer eventProcessorContainer;
+
+        static void Main(string[] args)
+        {
+            try
```

Comment:
We wanted to leave the file watcher code in here after discussion.  Again, leverage the bool to check if we should start, and then cancelToken on file change.